lib/blacklight/solr/search_builder_behavior.rb

```diff
@@ -190,6 +194,30 @@ def add_group_config_to_solr solr_parameters
       end
     end
 
+    def add_facet_paging_to_solr(solr_params)
+      return unless facet.present?
+
+      facet_config = blacklight_config.facet_fields[facet]
+
+      # Now override with our specific things for fetching facet values
+      solr_params[:"facet.field"] = with_ex_local_param((facet_config.ex if facet_config.respond_to?(:ex)), facet)
+
+      limit = if scope.respond_to?(:facet_list_limit)
+          scope.facet_list_limit.to_s.to_i
+        elsif solr_params["facet.limit"]
+          solr_params["facet.limit"].to_i
+        else
+          20
+        end
+
+      # Need to set as f.facet_field.facet.* to make sure we
+      # override any field-specific default in the solr request handler.
+      solr_params[:"f.#{facet}.facet.limit"]  = limit + 1
+      solr_params[:"f.#{facet}.facet.offset"] = ( blacklight_params.fetch(Blacklight::Solr::FacetPaginator.request_keys[:page] , 1).to_i - 1 ) * ( limit )
```

Comment:
Line is too long. [154/80]<br>Space found before comma.<br>Space inside parentheses detected.